public/js/report.js

```diff
@@ -259,7 +259,13 @@ var Report = React.createClass({
     },
 
     atLeastTwoHits: function () {
-        return this.state.queries.some(query => query.hits.length > 1);
+        var hit_num = 0;
+        _.each(this.state.queries, (function (obj) {
```

Comment:
Did eslint ask you to add `()` around the function callback? If not, please remove it.